src/shareddoc.js

```diff
@@ -396,41 +324,13 @@ export const messageListener = (conn, doc, message) => {
   }
 };
 
-export const deleteFromAdmin = async (docName, storage) => {
+export const invalidateFromAdmin = async (docName) => {
   const ydoc = docs.get(docName);
   if (ydoc) {
-    // If we still have the ydoc, set it to be empty.
-    // Note that it needs to contain at least one character to be picked up
-    // so setting it to a space.
-    ydoc.getMap('aem').set('svrinv', ' ');
-  }
-
-  const keys = await storage.get(['docstore', 'chunks', 'doc']);
-  const storedDoc = keys.get('doc');
-  if (storedDoc && storedDoc !== docName) {
-    // eslint-disable-next-line no-console
-    console.log('Mismatch between requested and found doc. Requested', docName, 'found', keys.get('doc'));
-    return false;
-  }
-
-  // eslint-disable-next-line no-console
-  console.log(
-    'Deleting storage for',
-    docName,
-    'containing',
-    keys.has('docstore') ? 'docstore' : `keys=${keys.chunks}`,
-  );
-  await storage.deleteAll();
-  return true;
-};
+    docs.delete(docName);
```

Comment:
Do you need to delete the doc from docs at this point? If you close all connections isn't that already happening in the closeCon call?

Reply:
You are right. That call wasn't needed. I removed it. Thanks!